src/js/handlers/registration.mjs

```diff
@@ -0,0 +1,29 @@
+import { register } from "../api/authentication.mjs";
+
+async function handleRegistrationEvent(name, email, password) {
+    try {
+        const response = await register(name, email, password);
+        
+        if (response.success) {
+            // Navigate back to login page
+            window.location.href = "/";
+        } else {
+            // TODO: Inform user of the error (i.e. Failed to create user profile)
+        }
```

Comment:
Again...code that is not being used should be removed - clean code is important.